1.12.0/csi-operator-1.12.0-RC2.yaml

```diff
@@ -871,27 +877,6 @@ roleRef:
   name: openebs-cstor-csi-registrar-role
   apiGroup: rbac.authorization.k8s.io
 
----
```

Comment:
Don't we need to revert the changes made in [commit](https://github.com/openebs/cstor-csi/pull/101/files#diff-672f2ef4bc468f4edd0828699604b2c9).

Reply:
Those are reverted from 1.12.x branch